server/TypeDBServer.java

```diff
@@ -68,14 +68,14 @@
     private static final Logger LOG = LoggerFactory.getLogger(TypeDBServer.class);
 
     protected final Factory factory;
-    protected final TypeDB typedb;
+    protected final TypeDB.DatabaseManager databaseMgr;
```

Comment:
Now that `RocksTypeDB` has been deleted, `databaseManager` is owned by `TypeDBServer` directly.